### DISCUSSION TITLE
After updating to the latest version of RocketChat, I can't login

### DISCUSSION THREAD

**Discussion by iabazi** [RocketChat/Rocket.Chat#30934] [Q&A]
I am having this error "User not found or incorrect password" after I updated my RocketChat hosted on DigitalOcean's droplet. My application wouldn't require the users to login to the chat, if they had the permission in the application generally, they could automatically be logged in RocketChat without ever having the need to put any credentials in the login page, instead the login page of RocketChat wouldn't show up at all.

Please, let me know how can I resolve this?

I looked and found that this problem has persisted for other people as well after making updates, but I couldn't really understand if they were able to fix it or not. I know some of them were talking about LDAP settings etc, I checked in my droplet's docker where I have installed RocketChat, and I don't have LDAP settings enabled. 

I changed the admin's password through the container in MongoDB, but that still didn't help in logging in. 

I would really appreciate if someone can help me get out of this problem.

**Comment by mremptypants** [comment]
Same for me too, did you find anything?

**Comment by reetp** [comment]
> I am having this error "User not found or incorrect password" after I updated my RocketChat hosted on DigitalOcean's droplet. My application wouldn't require the users to login to the chat, if they had the permission in the application generally, they could automatically be logged in RocketChat without ever having the need to put any credentials in the login page, instead the login page of RocketChat wouldn't show up at all.

Sounds like someone has written some bespoke code to make that happen as it is not normal behaviour with Rocket.Chat - you normally always need a user and password to login.

Some tips on asking for help, and the correct places to do so are here:

https://gist.github.com/reetp/a66149d5f060f260643a353ca7067a98

Please open this in the appropriate place.
